txgh/lib/txgh/utils.rb

```diff
@@ -64,6 +64,23 @@ def booleanize(obj)
           obj
       end
     end
+
+    def deep_symbolize_keys(obj)
```

Comment:
Ok, I did a cursory read-through of all the code in txgh and I don't think we're using any other activesupport-ish functionality (except `deep_symbolize_keys`). Txdb uses the activesupport inflector and accordingly depends on it in the gemspec.